pkgs/development/tools/convox/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, fetchurl, buildGoPackage }:
```

Comment:
I think it's more common or idiomatic to put `stdenv` instead of `lib` directly. Your `meta` then becomes:

``` nix
{
  meta = with stdenv.lib; {
    description = "Open-source PaaS";
    homepage = https://convox.com;
    license = licenses.asl20;
    maintainers = with maintainers; [ shosti ];
    platforms = with platforms; linux ++ darwin;
  };
}
```